language/move-lang/tests/move_check/parser/function_params_commas.move

```diff
@@ -0,0 +1,7 @@
+module M {
+    // Commas between function parameters are all optional.
+    f(p1: u64 p2: &u64) { }
```

Comment:
Can we fix this... This is definitely not intentional. 

I'm fine with waiting for another PR

Reply:
This is fixed as part of https://github.com/libra/libra/pull/1973